src/NUnitFramework/tests/TestUtils.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Threading;
+using NUnit.Framework.Internal;
+
+namespace NUnit.Framework
```

Comment:
Should this class be in the folder/namespace `TestUtilities` like the previous class?

Reply:
Sure, was 50/50 on this.

Reply:
Ugh, also forgot the license boilerplate.